adapters/coinzilla/coinzilla_test.go

```diff
@@ -0,0 +1,23 @@
+package coinzilla
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+const testsDir = "coinzillatest"
+const testsBidderEndpoint = "http://request-global.czilladx.com/serve/prebid-server.php"
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderAdpone, config.Adapter{
```

Comment:
I believe you mean `openrtb_ext.BidderCoinzilla` here

Reply:
Updated.